### ISSUE TITLE
output format,

### ISSUE THREAD

**Issue by cypriotxx** [SergeiKlimenko/Bible-downloader#1]
possible to have your script writing the books in this format where can be used in this script ?
https://github.com/digitalbiblesociety/browserbible/tree/master/app/content/bibles

**Comment by SergeiKlimenko** [comment]
Sorry, could you rephrase that? I'm not sure I understand your question.

**Comment by cypriotxx** [comment]
for example , the book that is scraped , can you make it to write the books like this so that books can be used in readers , this is ENG KJV bible translation , see the attached 
https://github.com/digitalbiblesociety/browserbible-bibles/blob/master/bibles/eng_kjv.zip
